src/main/java/net/snowflake/client/jdbc/SnowflakeBaseResultSet.java

```diff
@@ -134,7 +134,7 @@ protected void raiseSQLExceptionIfResultSetIsClosed() throws SQLException {
   @Override
   public Date getDate(int columnIndex) throws SQLException {
     raiseSQLExceptionIfResultSetIsClosed();
-    return getDate(columnIndex, (TimeZone) null);
+    return getDate(columnIndex, TimeZone.getDefault());
```

Comment:
is it a compatible change?

Reply:
I think this could be a BCR, because the behavior could change for other customers calling this. If we want to maintain behavior as before, I would have to suggest to change it back to null and tell the customer in [244](https://github.com/snowflakedb/snowflake-sdks-drivers-issues-teamwork/issues/244) to set JDBC_FORMAT_DATE_WITH_TIMEZONE=false

Reply:
Let's avoid a BCR - workaround should be fine to not break other integrations